humble/distribution.yaml

```diff
@@ -4765,6 +4765,21 @@ repositories:
       url: https://github.com/ros-planning/navigation_msgs.git
       version: humble
     status: maintained
+  nicla_vision_ros2:
+    doc:
+      type: git
+      url: https://github.com/ADVRHumanoids/nicla_vision_ros2
+      version: humble
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: TODO
+      version: TODO
```

Comment:
Are you only adding a doc and source entry for now? If so please delete the "release" section. `bloom-release` will create it for you when you release.